gamersnet_frontend/src/app.js

```diff
@@ -43,7 +44,8 @@ export default class App extends React.Component {
 		return (
 			<div>
 				<Route exact path = '/' component = {Home} />
-				<Route path = '/post' render = {(props) => <AddPost updateHeader = {this.updateHeader} {...props} />} />
+				<Route path = '/post' render = {(props) => <AddPost updateHeader = {this.updateHeader} {...props} />} /> 
+				<Route path = '/displayPosts' render = {(props) => <DisplayPosts updateHeader = {this.updateHeader} {...props} />} /> 
```

Comment:
Highlighting here for reference in another comment

Reply:
yes, I got what you were trying to say, we can remove this and that export default for displayPosts and Post after we merge your account_feature branch